### DISCUSSION TITLE
getting error while building spring boot maven project

### DISCUSSION THREAD

**Discussion by Rahulupadhyay17** [akash-coded/spring-framework#101] [Q&A]
![image](https://user-images.githubusercontent.com/98265104/170823205-d8a5e58f-5f07-44fb-97c0-24e02cb323ff.png)

@akash-coded i am getting error while building the project.

**Comment by Rahulupadhyay17** [comment]
![image](https://user-images.githubusercontent.com/98265104/170823337-baf88bd3-a8ad-4736-b9e8-27f4bd29c1f7.png)

@akash-coded and getting this while opening pom.xml

**Comment by Rahulupadhyay17** [comment]
i reinstalled eclipse and now it is working fine.

@akash-coded but i am not getting option to run application as spring boot app

**Comment by Rahulupadhyay17** [comment]
@akash-coded 
Its **rupadhyay**  from Feb cohort.

Sir, i have to install the spring tools 4 over and over again when i reopen Eclipse each time.
Does it work in same way or there is some issue with it?

Kindly help

**Reply by akash-coded** [reply]
@Rahulupadhyay17 Please download Eclipse from this link here: [Eclipse IDE](https://www.eclipse.org/downloads/packages/release/2022-03/r/eclipse-ide-enterprise-java-and-web-developers) for Enterprise Java and Web Developers

After that download the recommended plugins from Eclipse Marketplace, at least Spring Tools 4. Restart Eclipse. Your installation should be working fine after that.
